src/loaders/gtdb/gtdb_genome_stats_loader.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import hashlib
+
+import pandas as pd
+
+import gtdb_genome_stats_helper as helper
+from gtdb_loader_helper import convert_to_json
+
+"""
+PROTOTYPE
+
+Prepare GTDB genome statistics data in JSON format for arango import.
+
+This script will first parse genome features from the GTDB metadata files. Those features should be put into
+EXIST_FEATURES. 
+
+It will then try to compute other(non-existing) features. The name of non-existing features should be put into 
+NON_EXIST_FEATURES. Each non-existing features should have a corresponding function defined in gtdb_genome_stats_helper.
+NOTE: the function name should be exactly the same as the non-existing feature name.
```

Comment:
👍  it's done on the helper module.

Reply:
Oops, missed that, sorry. Could it be a bit more specific though, something like

> Methods accept a pandas dataframe and should return a pandas series containing only the computed genome feature. It will then be appended to the result frame.

Reply:
👍

Reply:
:+1: